test/sum.test.js

```diff
@@ -0,0 +1,14 @@
+describe('sum', function () {
```

Comment:
Not sure these are worth keeping in the library - but at least added tests for coverage stats.